standalone/src/models/custom-application/custom-application-permission/custom-application-permission-draft/builders.spec.ts

```diff
@@ -0,0 +1,15 @@
+import type { TCustomApplicationPermissionDraftGraphql } from './types';
+import { CustomApplicationPermissionDraftGraphql } from './index';
+describe('builder', () => {
+  it('GraphQL model', () => {
+    const graphqlModel =
+      CustomApplicationPermissionDraftGraphql.random().buildGraphql<TCustomApplicationPermissionDraftGraphql>();
+
+    expect(graphqlModel).toEqual(
+      expect.objectContaining({
+        name: expect.any(String),
+        oAuthScopes: expect.any(Array),
```

Comment:
Can we expect that `oAuthScopes` is an array of strings, please? 🙏

Reply:
fixed in https://github.com/commercetools/test-data/pull/827/commits/845ec4435d340dbd6a28c2debcdbf2985e463201